src/main/java/com/google/cloud/genomics/dataflow/readers/ReadStreamer.java

```diff
@@ -77,11 +79,11 @@ public static StreamReadsRequest getReadRequests(String readGroupSetId) {
     final Iterable<Contig> contigs = Contig.parseContigsFromCommandLine(references);
     return FluentIterable.from(contigs)
         .transformAndConcat(new Function<Contig, Iterable<Contig>>() {
-            @Override
-            public Iterable<Contig> apply(Contig contig) {
-              return contig.getShards(100000000L);
-            }
-          })
+          @Override
+          public Iterable<Contig> apply(Contig contig) {
+            return contig.getShards(5000000L);
```

Comment:
I think Nicole and Dion previously discussed issues with hardcoding it, but maybe having a final constant and just manipulating that might be better.